### PR TITLE
Clarified API publishing rules

### PR DESCRIPTION
**Pull request by tkrop**

### CODE REVIEW

File: chapters/api-operation.adoc [hunk 1]
```diff
@@ -9,11 +9,14 @@ APIs. While this is optional for internal APIs, i.e. APIs marked with the
...
-OpenAPI specifications are published via deployment artifacts by the API owner,
-as described in the internal https://docs.google.com/a/zalando.de/document/d/1WBpRHp1DAtz7Pfjt0QwPuZNR1e70APRqUlCCby3JiOM/edit?usp=sharing[narrative].
-We prefer the deployment artifact-based method over the past (now legacy)
-`.well-known/schema-discovery` service endpoint-based publishing process. However,
-we currently support this way for backward compatibility reasons.
+An API is published by copying its **OpenAPI specification** into the reserved
+**/zalando-apis** directory of the **deployment artifact** used to deploy the
+provisioning service. The directory must only contain **self-contained YAML**
+**files** that each describe one API (see 
+https://docs.google.com/a/zalando.de/document/d/1WBpRHp1DAtz7Pfjt0QwPuZNR1e70APRqUlCCby3JiOM[internal narrative]
```
Comment (dehora) [inline chapters/api-operation.adoc]: Can this link be removed? The guidelines are public and referencing internal documents doesn't work well.

**Comment by duergner** [comment]
:+1:

File: chapters/api-operation.adoc (hunk 1 above)
Comment (tkrop) [reply chapters/api-operation.adoc]: Not our current strategy.

**Comment by tfrauenstein** [comment]
👍

**Comment by ePaul** [comment]
:+1:

**Pull request merged**